view/base/web/js/action/account-add-exemption.js

```diff
@@ -0,0 +1,84 @@
+define(['jquery', 'uiComponent', 'Magento_Ui/js/modal/modal'], function (jQuery, Component, modal) {
+    return Component.extend({
+        defaults: {
+            template: 'ClassyLlama_AvaTax/action/account-add-exemption',
+            exemptionZone: '',
+            availableExemptionZones: [],
+            showSdkView: false,
+            certificateUploadSuccess: false,
+            sdkParameters: {}
+        },
+
+        initialize: function initialize() {
+            this._super();
+
+            this.observe(['showSdkView', 'exemptionZone', 'certificateUploadSuccess']);
+            this.onCertificateComplete = this.onCertificateComplete.bind(this);
+            this.onSdkLoad = this.onSdkLoad.bind(this);
+
+            this.sdkParameters = jQuery.extend({
+                // Include if cert is a renewal?
```

Comment:
Remove this comment (unless you think it ought to be left in?)

Reply:
Honestly, i still don't have much idea on how the SDK works. I'm sure that comment was relevant at one point, but since i cannot confirm or deny that claim, I'd rather leave it in, just in case it makes sense to someone else down the line.